pandas/core/groupby.py

```diff
@@ -116,16 +116,34 @@ def _groupby_function(name, alias, npfunc, numeric_only=True,
     def f(self, **kwargs):
         if 'numeric_only' not in kwargs:
             kwargs['numeric_only'] = numeric_only
+        if 'skipna' in kwargs:
+            checknan = not kwargs.pop('skipna')
+        else:
+            checknan = not skipna
+
         self._set_group_selection()
         try:
-            return self._cython_agg_general(alias, alt=npfunc, **kwargs)
+            result = self._cython_agg_general(alias, alt=npfunc, **kwargs)
         except AssertionError as e:
             raise SpecificationError(str(e))
         except Exception:
             result = self.aggregate(lambda x: npfunc(x, axis=self.axis))
-            if _convert:
-                result = result._convert(datetime=True)
-            return result
+
```

Comment:
hmm, I think we actually should add a parameter to the cython routines. it would go in ``pandas/_libs/algos_groupby_helper.pxi.in``; you can call it ``skipna``.